docs/internals/ARCHITECTURE.md

```diff
@@ -468,6 +468,67 @@ This high-quality time implementation is than utilized not only for baseness log
 also plays a key role in the internal implementation. Every object in TigerBeetle has a globally
 unique `u64` creation timestamp, which plays the role of synthetic primary key.
 
+### Direct IO
+
+TigerBeetle bypasses operating system's page cache and uses Direct IO. That is, normally, when an
+application writes to a file, the operating system only updates the in-memory cache, and the data
+gets to disk later. This is a good default for the vast majority of the applications. TigerBeetle is
+an exception --- it instructs the operating system to read directly from the disk, and write directly
+to the disk, bypassing any caches (please refer to this
+[excellent article](https://transactional.blog/how-to-learn/disk-io) for the overview of the
+relevant OS APIs).
+
+Bypassing page cache is required for correctness. While operating systems provide `fsync` API to
```

Comment:
```suggestion
Bypassing the page cache is required for correctness. While operating systems provide `fsync` API to
```

(but, this one reads OK as "bypassing page cache" too I think, just later we refer to "TigerBeetle doesn't require the OS to provide a page cache" which could also be "TigerBeetle doesn't require the OS to provide page cache")

Reply:
dropped both articles!